test/int/testSetup.ts

```diff
@@ -13,20 +13,33 @@ import {DebugClient} from 'vscode-debugadapter-testsupport';
 
 const DEBUG_ADAPTER = './out/src/chromeDebug.js';
 
-function patchLaunchArgs(launchArgs: any): void {
+var testBreakOnLoadStrategy: string;
+
+function formLaunchArgs(launchArgs: any): void {
     launchArgs.trace = 'verbose';
     launchArgs.disableNetworkCache = true;
 
     // Start with a clean userDataDir for each test run
     const tmpDir = tmp.dirSync({ prefix: 'chrome2-' });
     launchArgs.userDataDir = tmpDir.name;
+    if (testBreakOnLoadStrategy) {
+        launchArgs.breakOnLoadStrategy = testBreakOnLoadStrategy;
+        testBreakOnLoadStrategy = undefined;
+    }
+}
+
+function patchLaunchArgs(launchArgs: any): void {
+    formLaunchArgs(launchArgs);
 }
 
 export const lowercaseDriveLetterDirname = __dirname.charAt(0).toLowerCase() + __dirname.substr(1);
 export const PROJECT_ROOT = path.join(lowercaseDriveLetterDirname, '../../../');
 export const DATA_ROOT = path.join(PROJECT_ROOT, 'testdata/');
 
-export function setup(port?: number) {
+export function setup(port?: number, breakOnLoadStrategy?: string) {
```

Comment:
Instead of having a special property for breakOnLoadStrategy, could you extend it to allow any launch config prop to be overridden? ie make this an object and apply all the props above.